From 513840863dccc4665ba5e2f8790630ce5a8a9913 Mon Sep 17 00:00:00 2001
From: Tiberiu Breana <andrei-tiberiu.breana@nxp.com>
Date: Thu, 16 Mar 2017 11:30:17 +0200
Subject: [PATCH 1582/5242] MLK-14426: ARM: dts: imx6ul-evk-btwifi: Remove
 imx6ul-evk-murata-v2 function node

commit  cd3c094b9b8bac13e7a7da55a75a6ee1fe1294e9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Removed the unused imx6ul-evk-murata-v2 function node which was
causing a kernel panic at boot time.

Signed-off-by: Tiberiu Breana <andrei-tiberiu.breana@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi |   10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi b/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
index 012f4d6..d5b6c19 100644
--- a/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
+++ b/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
@@ -32,12 +32,10 @@
 };
 
 &iomuxc {
-	imx6ul-evk-murata-v2 {
-		pinctrl_wifi: wifigrp {
-			fsl,pins = <
-				MX6UL_PAD_SNVS_TAMPER1__GPIO5_IO01 0x03029
-			>;
-		};
+	pinctrl_wifi: wifigrp {
+		fsl,pins = <
+			MX6UL_PAD_SNVS_TAMPER1__GPIO5_IO01 0x03029
+		>;
 	};
 };
 
-- 
1.7.9.5

